apps/web/cypress/plugins/index.ts

```diff
@@ -1,3 +1,5 @@
+import { IntegrationService } from '@notifire/testing/src/integraion.service';
```

Comment:
Could you add the IntegrationService to the export barrels so the import will be:
```suggestion
import { IntegrationService } from '@notifire/testing';
```

Reply:
This causes some issues when deployed in a docker env later on.